Backend/Models/Position.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Backend
+{
+    public partial class Position
+    {
+        public bool IsWithinViewport(Viewport viewport)
+        {
+            // naive implementation, ignores edge cases
```

Comment:
Nitpick but there seems to be some form of defacto standard to use SouthWest and NorthEast points as bounds for viewports in maps. at least that is how MapBox and Google Maps deal with bounds.
It might be good to stick to that standard instead of lng1 / lng2 etc

Reply:
I've updated Viewport class and related properties to follow this standard.